integrations/pagerDuty.js

```diff
@@ -132,73 +139,81 @@ async function createIncident(authData, user) {
  * @param {object} authData The user's entered auth data
  * @param {object} user     The current SDK user
  */
-async function searchIncidents(authData, user) {
+async function searchIncidents(authData) {
   // Init our PD client
   const { pagerDutyKey } = authData
   pd.initializePagerDuty(pagerDutyKey)
 
   // Get query information from the user
-  const query = await ux.prompt(searchIncidentsPrompt)
+  const query = await getSearchQuery()
 
-  sdk.log('')
+  await ux.print('')
   ux.spinner.start('Retrieving incidents')
   const incidents = await pd.getIncidents(query)
   ux.spinner.stop('Done!\n')
 
   // Nothing found; return early
   if (!incidents.length) {
-    sdk.log(ux.colors.blue('No incidents found!'))
+    await ux.print(ux.colors.blue('No incidents found!'))
     return incidents
   }
 
-  sdk.log(ux.colors.magenta('\nWe found the following incidents:\n'))
-  incidents.map(printIncident)
+  await ux.print(ux.colors.magenta('\nWe found the following incidents:'))
+  const incidentsStr = incidents.map(printIncident)
```

Comment:
Just a note - resolve after you've read it.

My instinct here was that we should combine it into one big string and print that to slack - but then I realized it probably wouldn't work because it could be too long to fit in one Slack message. Something to definitely keep in mind!

Reply:
Can you try it in slack and see if it breaks currently?